domains/_github-pages-challenge-shadowisdev.shadowalker.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "shadowisdev",
+        "email": "shadowisdev@gmail.com"
+    },
+    "record": {
+        "TXT": ""
```

Comment:
Please put your GitHub Pages verification TXT here.

Reply:
Ohk i will

Reply:
i didn't get any txt record so removed that file, you can merge it now. i will add that file when i get a mail like that.